environments/production/terraform.yml

```diff
@@ -281,14 +282,15 @@ servers:
     volume_size: 160
     group: "kafka"
     os: trusty
-
-  - server_name: "pgproxy1-production"
```

Comment:
I don't see this getting removed elsewhere in the PR

Reply:
it removed from the inventory file in other PR while replacing old with new pgproxy server

https://github.com/dimagi/commcare-cloud/pull/3343

Reply:
originally removed here https://github.com/dimagi/commcare-cloud/pull/3343